docs/docs/visual-testing-with-storybook.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Visual Testing with Storybook
+---
+
+Knowing your components look as intended in every permutation is not only a great way to test them visually, but also provides "living documentation" for them. This makes it easier for teams to 1) know what components are available to them in a given project and 2) what props those components accept and what all of the states of that component are. As your project grows over time having this information available will be invaluable.
+
+## Setting up your environment
+
+In order to set up Storybook you will need to install a few dependencies and then do some custom configuration to make Storybook work with your Gatsby environment. The first thing is to get the Storybook CLI installed if you don't have it already.
```

Comment:
This can be less wordy, perhaps:
```suggestion
To set up Storybook you need to install dependencies and do some custom configuration. First, install the Storybook CLI.
```